src/components/shared/InfoPopover/MemberInfoPopover/MemberInfoPopover.tsx

```diff
@@ -40,63 +42,63 @@ const MemberInfoPopover = ({
     profile: { walletAddress },
   } = user;
 
-  const { data: nativeTokenAddressData, loading: loadingNativeTokenAddress } =
-    useColonyNativeTokenQuery({
-      variables: { address: colonyAddress },
-    });
+  // const { data: nativeTokenAddressData, loading: loadingNativeTokenAddress } =
+  //   useColonyNativeTokenQuery({
+  //     variables: { address: colonyAddress },
+  //   });
 
-  const { data: userReputationData, loading: loadingUserReputation } =
-    useUserReputationForTopDomainsQuery({
-      variables: { address: walletAddress, colonyAddress },
-    });
+  // const { data: userReputationData, loading: loadingUserReputation } =
+  //   useUserReputationForTopDomainsQuery({
+  //     variables: { address: walletAddress, colonyAddress },
+  //   });
 
   const allUserRoles = useTransformer(getAllUserRoles, [colony, walletAddress]);
-  const { data: userBalanceData, loading: loadingUserBalance } =
-    useUserBalanceWithLockQuery({
-      variables: {
-        address: walletAddress,
-        tokenAddress:
-          nativeTokenAddressData?.processedColony.nativeTokenAddress || '',
-        colonyAddress,
-      },
-      /*
-      The fetchPolicy here is "no-cache" because otherwise the result would be stored
-      in the cache and then the current user's balance would change (in the other instances
-      where the current user balance is obtained using this query) to the one that it's being
-      displayed in the popover.
-
-      It could also happen in reverse, the current user's balance could show up here instead
-      of the balance of a "second person" user.
-
-      Basically, all sorts of shenanigans happen when this result is stored on the cache.
-    */
-      fetchPolicy: 'no-cache',
-    });
-
-  if (
-    loadingNativeTokenAddress ||
-    loadingUserReputation ||
-    loadingUserBalance
-  ) {
-    return (
-      <div className={`${styles.main} ${styles.loadingSpinnerContainer}`}>
-        <SpinnerLoader
-          appearance={{
-            theme: 'primary',
-            size: 'medium',
-            layout: 'horizontal',
-          }}
-        />
-      </div>
-    );
-  }
-  const nativeToken = userBalanceData?.user.userLock.nativeToken;
-  const userLock = userBalanceData?.user.userLock;
-  const inactiveBalance = BigNumber.from(userLock?.nativeToken?.balance || 0);
-  const lockedBalance = BigNumber.from(userLock?.totalObligation || 0);
-  const activeBalance = BigNumber.from(userLock?.activeTokens || 0);
-
-  const totalBalance = inactiveBalance.add(activeBalance).add(lockedBalance);
+  // const { data: userBalanceData, loading: loadingUserBalance } =
```

Comment:
Again, please check with @Julianahlk so that this aligns (and maybe you don't run into conflicts) with the changes she's doing with the avatars / popovers